translation/en/RedirectedResources/assets/abdata/adv/scenario/c34/20/2/translation.txt

```diff
@@ -1,13 +1,13 @@
-//「この国のコト、スコシ分かてキマシタ。ケド、アナタのコト、トテモよく分かりマシタ」=
-//「え？」=
-//「ワタシ、アナタのコト、好きデス♡　オトコノコ、トシテ、愛、感じマス♡」=
-//「えええっ！？　な、なんで！？」=
-//「ナンデ？　Ｏｈ，ｒｅａｓｏｎデスネ？　ニッポン語出来ないワタシに、イッパイ親切くれマシタデスカラ」=
-//「Ｃｌａｓｓｍａｔｅ、先生、ニッポンの人、皆サン親切デスケド、距離アリマス。言葉の壁タイヘンデース」=
-//「デモ、アナタはイッパイｃｏｍｍｕｎｉｃａｔｉｏｎして、タクサンａｐｐｒｏａｃｈくれマシタ。アナタダケ、ｓｐｅｃｉａｌデシタ♡」=
-//「そんな風に思ってくれるなら、話しかけた甲斐があったよ。それにしたっていきなり好きだなんて、ビックリしたよ」=
-//「フフ、ワタシが育った国、オンナノコ、ミンナａｇｇｒｅｓｓｉｖｅデシタ。アナタ、ワタシのｆｉｒｓｔ　ｌｏｖｅデース」=
-//（ファーストラブって初恋だよな。ちゃんと俺の人となりを見て好きになってくれたのは嬉しいけど、展開が一段飛ばしじゃないか！？）=
-//「アッ！　イマ、恋人になてほしいと違いマスヨ？　コレカラ、モット仲良ししたいいう意味デスノデ、ヨロシクお願いシマス♡」=
-//「[H]の気持ちはちゃんと伝わったよ。これからもよろしくね」=
-//（とは言ったけど、これからは話すだけでドキドキしちゃいそうだな）=
+﻿「この国のコト、スコシ分かてキマシタ。ケド、アナタのコト、トテモよく分かりマシタ」=I feel like I understand a bit what Japan is about now. But, I understand you a whole lot more.
+「え？」=Eh?
+「ワタシ、アナタのコト、好きデス♡　オトコノコ、トシテ、愛、感じマス♡」=I love you♡ I feel love for you, as a woman♡
+「えええっ！？　な、なんで！？」=EEEEEEH!? W-Why!?
+「ナンデ？　Ｏｈ，ｒｅａｓｏｎデスネ？　ニッポン語出来ないワタシに、イッパイ親切くれマシタデスカラ」="Why?" Oh, you're asking about the reason? It's because you showed me so much kindness, a girl who can't into Japanese.""
```

Comment:
It's odd when existing English words don't exist in the translation, even when it's broken English. Also this line has funky quoting.

Reply:
Translator got leave to take some creative freedoms in this personality, as a straight translation would be really awkward, thus some weird sentences, but a straight translation is even more awkward